src/serde/ser.rs

```diff
@@ -0,0 +1,328 @@
+use core::result::Result;
```

Comment:
Why is this needed, `Result` is part of the implicit prelude import.

Reply:
removed